docs/instruction_for_testing.md

```diff
@@ -0,0 +1,82 @@
+# Running tests before release
```

Comment:
TODO - change filename to instructions_for_running_integration_tests
Title How to run full integration test suite before release